project_profitability_fields/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    "name": "Project Profitability Fields",
+    "summary": """
+        Add custom fields to track project profitability."
+    """,
+    "author": "Solvos",
+    "license": "AGPL-3",
+    "version": "15.0.0.0.0",

```

Comment:
Wrong version, see https://github.com/solvosci/slv-docker-odoo-prod/wiki/Odoo-SLV-contribution-brief-guidelines#24-manifest for more details